examples/react/urql-string-document-mode/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I think urql is not a good example because it actually requires the AST at runtime anyways 🤔 .
We can add or modify a react-query or similar example that does not require the AST.
What do you think?

Reply:
> I think urql is not a good example because it actually requires the AST at runtime anyways 

Does it, though? It accepts both string and AST and seems to work fine (I modified the original urql example). Unless I'm missing something.

Regarding the `react-query` example — it currently uses `@graphql-tools/executor-http`, but I could replace it with `fetch`.

Reply:
yeah, urql uses graphql internally and thus will just parse the string to the ast at runtime

Reply:
yeah, I am fine with replacing executor HTTP in react-query!